src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/locations/availability/DateTimeAvailabilityService.kt

```diff
@@ -70,6 +92,10 @@ class DateTimeAvailabilityService(
     )
   }
 
+  private fun isTheSame(request: DateTimeAvailabilityRequest, location: Location, prisonAppointment: PrisonAppointment) = run {
+    location.dpsLocationId == prisonAppointment.prisonLocationId && prisonAppointment.startTime == request.startTime && prisonAppointment.endTime == request.endTime
+  }
```

Comment:
Should the date also be included in the check?

Reply:
🤦‍♂️